Translations update from Weblate

weblate-fpf opened the pull request:
Translations update from [Weblate](https://weblate.securedrop.org) for [SecureDrop/SecureDrop](https://weblate.securedrop.org/projects/securedrop/securedrop/).


It also includes following components:

* [SecureDrop/desktop](https://weblate.securedrop.org/projects/securedrop/desktop/)



Current translation status:

![Weblate translation status](https://weblate.securedrop.org/widgets/securedrop/-/securedrop/horizontal-auto.svg)

cfm commented:
I think we've got it.  #6954 hereby achieves #6232—although there are some anticlimactic follow-up tasks there that prevent closing the epic just yet.

legoktm commented:
Yay! Just out of curiosity, will there always be a bunch of "Update translation files" commits or is that just because this is the first go? (And if it's the latter, should it be squashed down before merging?)

cfm commented:
On Wed, Oct 11, 2023 at 09:23:40PM -0700, Kunal Mehta wrote:
> Yay! Just out of curiosity, will there always be a bunch of "Update
> translation files" commits or is that just because this is the first
> go? (And if it's the latter, should it be squashed down before
> merging?)

Yes, this first one is noisy because of some drift in headers, sorting,
etc. that I either couldn't suppress or took the opportunity to
normalize.  Weblate itself is squashing down to one commit per language
for easier review.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Icelandic)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Icelandic)

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Sveinn í Felli <sv1@fellsnet.is>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/is/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/is/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Translated using Weblate (Greek)

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Greek)

Currently translated at 99.7% (337 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Adrian <pappasadrian@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: norhorn <norhorn@gmail.com>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/el/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Catalan)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Catalan)

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Joan Montané <joan@montane.cat>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/ca/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/ca/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Portuguese (Portugal))

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: deeplow <deeplower@protonmail.com>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/pt_PT/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Translated using Weblate (Czech)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Czech)

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: 1000101 <b1000101@pm.me>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/cs/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/cs/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Reader <isabellinacrew@tuta.io>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/zh_Hans/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/zh_Hans/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Swedish)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Jonas Waga <jonas.waga@schibsted.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/sv/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/sv/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Turkish)

Currently translated at 100.0% (10 of 10 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Alper Gunay <aalpergunay@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/tr/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: SecureDrop/desktop
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Esperanto)

Currently translated at 100.0% (10 of 10 strings)

Added translation using Weblate (Esperanto)

Translated using Weblate (Esperanto)

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Cory Francis Myers <cory@freedom.press>
Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: anonymous <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/eo/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/eo/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Translated using Weblate (Slovak)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Slovak)

Currently translated at 100.0% (338 of 338 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: 1000101 <b1000101@pm.me>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/sk/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/sk/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Merge remote-tracking branch 'origin/develop' into develop

zenmonkeykstop commented on install_files/ansible-base/roles/tails-config/templates/locale/eo/LC_MESSAGES/messages.po:
```diff
@@ -0,0 +1,40 @@
+
+#. Icon
+msgid "{{ tails_config_securedrop_dotfiles }}/securedrop_icon.png"
+msgstr "[!!!!!{{ tails_config_securedrop_dotfiles }}/securedrop_icon.png!!!!!]"
```
This is a path, not a translatable string. The substitution above will break in the desktop files, and it's not clear to me that it should be present at all.

cfm replied:
Yep.  `BabelGladeExtractor` extracts this because some desktop apps might well localize (not translate) their desktop icons for different audiences.  For us, it's a no-op except for this automatic translation by Weblate, which after #7008 will never be used.

zenmonkeykstop reviewed (approved):
LGTM, with one nit in the eo desktop files that can be addressed separately.
